src/main/java/im/status/keycard/KeycardApplet.java

```diff
@@ -1225,13 +1235,24 @@ private void generateKey(APDU apdu) {
     // Save seed exportability (this also indicates the seed as valid)
     masterSeedStatus = exportability; // Only save seed exportability after seed successfully loaded
 
+    // Build response [KeyUID, (optional) Seed]
+    short off = SecureChannel.SC_OUT_OFFSET;
```

Comment:
Maybe wrap the `JCSystem.beginTransation` up here so if either function fails, so does the atomic transaction.

Thoughts?

Reply:
I can do that

Reply:
I force-pushed the change. Refresh to see it.